docs/diagrams/DeleteActivityDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+start
+:User enters a DeleteCommand;
+:DeleteCommandParser parses for valid format;
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if (Delete command is valid?) then (yes)
```

Comment:
The recommended format in the text book is that the diamond node should be empty inside so can perhaps do 
```
if () then ([Delete command is valid])
...
else ([else])
```

instead

Reply:
Oh, thanks, I didn't know that